translations/bnpchs-final-2019-Sesi-Belajar.html

```diff
@@ -0,0 +1,70 @@
+<!-- Title: Study Session-->
+<h3>Description</h3>
+
+<p style="text-align: justify;">
+  Feli is a college student who recently took a part-time job as a teacher. In this part-time job, Feli has a task to teach and improve the grade of $$$N$$$ students. Initially, the $$$i$$$-th student has an academic grade of $$$X_i$$$. To finish his assignment, Feli must make each student's academic grade to $$$M$$$.
+</p>
+
+<p style="text-align: justify;">
+  In a single study session, Feli can teach multiple students at once, and every student who attends the session gets it's academic grade increases by one. If the grade of a student reaches $$$M$$$, then the student feels that he/she has mastered the lesson and won't attend the study session again. 
+</p>
+
+<p style="text-align: justify;">
+  The friendship of the students is very close, in a study session where student $$$L$$$-th and $$$R$$$-th attend the session, every student with index between $$$L$$$ and $$$R$$$(inclusive) must come too. 
+</p>
+
+<p style="text-align: justify;">
+  Help Feli by telling her the minimum study session needed to finish the task, so that she can return to her holiday as soon as possible.
```

Comment:
```suggestion
  Help Feli by telling her the minimum study session needed to finish the task, so that she can return to her vacation as soon as possible.
```